<?xml version="1.0"?>
<launch>
    <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}"/>

    <arg name="argID"       default="0"/>
    <arg name="argVehicle"  default="heifu"/>
    <arg name="argNamespace"   default="$(arg argVehicle)$(arg argID)"/>

    <arg name="argNodeRate"         default="100.0"/>
    <arg name="argPubTopicSetpoint" default="/local_position"/>


    <node name="priority_manager_node" type="priority_manager_node" pkg="priority_manager" output="screen" ns="$(arg argNamespace)" required="true" args="debug">
        <rosparam param="entries" file="$(find priority_manager)/config/Entries.yaml" subst_value="True" />

        <param name="paramNodeRate"         type="double" value="$(arg argNodeRate)" />
        <param name="paramPubTopicSetpoint" type="string" value="$(arg argPubTopicSetpoint)" />


paramPubTopicSetpoint
    </node>
</launch>
